<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>801</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(16,30,41,240);</string>
   </property>
   <widget class="QLabel" name="welcomelabel">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>40</y>
      <width>332</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 36pt &quot;Calibri&quot;;
color:rgb(255,255,255);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Signup</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="infolabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>380</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 18pt &quot;Calibri&quot;;
color:rgb(255,255,255);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Create a new account</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>720</y>
      <width>300</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 18pt &quot;Calibri&quot;;
color:rgb(255,255,255);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_line">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>290</y>
      <width>310</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_line">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>400</y>
      <width>310</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>250</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Calibri&quot;;
color:rgb(255,255,255);
border-radius:10px;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>360</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Calibri&quot;;
color:rgb(255,255,255);
border-radius:10px;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup_button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>610</y>
      <width>300</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 18pt &quot;Calibri&quot;;
color:rgb(255,255,255);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Signup</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>570</y>
      <width>390</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 16pt &quot;Calibri&quot;;
color:red;
border-radius:10px;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>470</y>
      <width>300</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Calibri&quot;;
color:rgb(255,255,255);
border-radius:10px;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirm_line">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>510</y>
      <width>310</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>680</y>
      <width>300</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Calibri&quot;;
color:rgb(255,255,255);
border-radius:10px;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Already have an account?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
